src/utils/httpHeaders.ts

```diff
@@ -0,0 +1,14 @@
+import { APIGatewayProxyResult } from "aws-lambda";
```

Comment:
Not sure on the naming convention for this one either, should addHttpHeaders being a method in HttpHeaders class maybe? What are your thoughts?